### PR TITLE
Update casa urls

### PR DESCRIPTION
**Pull request by psibi**

### CODE REVIEW

**Review by snoyberg — APPROVED** [review]
FYI @mpilgrem this is a DevOps change on our side, trying to put the URLs where they belong.

**Comment by psibi** [comment]
Let's defer merging this till couple of days to see how the new deployment goes on.

File: package.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ dependencies:
 - filelock
 - generic-deriving
 - hackage-security
-- hpack >= 0.35.1
+- hpack
```
Comment (mpilgrem) [inline package.yaml:44]: I think the lower bound on `hpack` is needed, because of how `pantry` handles 'pretty' Hpack error messages.

**Review by mpilgrem — CHANGES_REQUESTED** [review]
I think we should retain the lower bound on `hpack`. Other than that, looks good to me.

**Comment by mpilgrem** [comment]
@psibi, I rebased on `master`. The (different) lower bound on `hpack` in `master` is now due to the move to `crypton`. Has enough time since deployment now passed to allow a merge?

**Comment by psibi** [comment]
@mpilgrem Yes, the build is stable now. The reason for the lower bound is because stackage server uses older packages (including cryptonite). That being said - I think this PR is good to merge. The stackage server will be using a different branch of pantry untill we fix it properly: https://github.com/fpco/stackage-server/pull/321/files#diff-0de01f189d384bfbe69e806ef70b1a425b27394dbe2a9d0e62ffd010dda86c03R13

**Comment by psibi** [comment]
Thanks for rebasing it, please merge it whenever you think is appropriate.

**Pull request merged**